web-ui/src/pages/TeamSkillReportPage.css

```diff
@@ -0,0 +1,72 @@
+.active {
```

Comment:
You need to do some prefixing on these names. These are too generic and will eventually have clash if they don't already. Either restrict them to only applying to things that are inside this component (set a class at the top level element and include it in your match patterns in this file) or make them very unlikely to clash (prefix the classnames themselves).